stable/jenkins/README.md

```diff
@@ -73,8 +73,8 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `Master.JMXPort`                  | Open a port, for JMX stats           | Not set                                                                      |
 | `Master.ExtraPorts`               | Open extra ports, for other uses     | Not set                                                                      |
 | `Master.OverwriteConfig`          | Replace config w/ ConfigMap on boot  | `false`                                                                      |
-| `Master.HostName`                 | Enables chart ingress, ingress host path      | Not set                                                                         |
-| `Master.Ingress.ApiVersion`      | Ingress api version                  | Not set                                                                         |
+| `Master.HostName`                 | Ingress host path      | Not set                                                                         |
```

Comment:
I would even go as far as moving the HostName field into ingress.
What do you think?

Reply:
As an outsider I would say it makes perfect sense, but I've no context as to why it lived outside that block in the first place :) Perfectly happy to move it though...at what point does this affect the versioning changes btw (this is definitely a breaking change, no?)

Reply:
you are right. this is why i suggested to bump the minor version instead of the patch.

Reply:
and as far as breaking changes go, i think that we are good to go as long as everything is well documented.